src/core.c/CompUnit/Repository/Locally.pm6

```diff
@@ -64,6 +65,12 @@ role CompUnit::Repository::Locally {
 
     # stubs
     method short-id(CompUnit::Repository::Locally:D: --> Str:D) { ... }
+
+    method normalize-path($path) {
+        (nqp::istype($path, IO::Path)
+            ?? $path.absolute
+            !! $path.Str.IO.absolute).IO.resolve.relative($.abspath)
```

Comment:
I don't think this method should be exposed - exposing its API doesn't really give users any benefit. It is only used in one spot, so it sort of feels like it should just be some helper private subroutine.

`$path.Str.IO.absolute` should probably just be `$path.IO.absolute`.